roles/contiv_network/tasks/main.yml

```diff
@@ -41,13 +29,20 @@
 - name: install contivctl
   shell: source /etc/profile.d/00golang.sh && go get github.com/contiv/contivctl
 
-- name: install docker-compose
-  shell: pip install docker-compose
+- yum: name=python-pip
 
-- name: install contiv-compose
-  shell: >
-     source /etc/profile.d/00golang.sh && \
-     go get github.com/contiv/libcompose && \
-     cd $GOPATH/src/github.com/contiv/libcompose && \
-     make binary && \
-     ln -s `pwd`/bundles/libcompose-cli_linux-amd64 $GOPATH/bin/contiv-compose
+- pip: name=docker-compose state=latest extra_args=--upgrade
+
+- name: copy environment file for netmaster
```

Comment:
this is a minor nit. I think as part of edits to this file the block for netmaster configuration got moved to the end of file starting from this line onwards. We should move it back to it's original position because the block that installs compose and other stuff above has a comment that things there need to be moved out of this file. So a reader in future might get an idea that even netmaster needs to be moved out of this file which can cause confusion :)